src/default-directory-provider.coffee

```diff
@@ -26,7 +26,7 @@ class DefaultDirectoryProvider
 
     # TODO: Stop normalizing the path in pathwatcher's Directory.
     directory = new Directory(directoryPath)
-    if protocol?
+    if host
```

Comment:
Shouldn't this be host?  i.e. only if not a URL given it is treat it as a plain directory.